src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,17 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1030004
+name: Mollaret’s meningitis
+def: "A rare clinical disorder characterized by at least three recurrent episodes of meningitis associated with spontaneous recovery with or without antiviral therapy. Herpes simplex virus type 2 has frequently been implicated in the setting of this illness. RBLM is rare with prevalence rates estimated at 2 per 100,000 persons." [https://orcid.org/0000-0001-9310-0163, PMID:36304598]
+synonym: "RBLM" EXACT ABBREVIATION [PMID:36304598]
+synonym: "recurrent benign lymphocytic meningitis" EXACT [https://orcid.org/0000-0001-9310-0163, PMID:36304598, PMID:37284381]
+xref: GARD:10868 {source="https://orcid.org/0000-0001-9310-0163", source="MONDO:equivalentTo"}
+is_a: MONDO:0007015 {xref="PMID:36304598"} ! viral meningitis
```

Comment:
we should use "source" instead of "xref" for SubClassOf annotation

Reply:
fixed